cluster/examples/kubernetes/nfs/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-nfs
+provisioner: nfs.rook.io/nfs
```

Comment:
what would this look like if we started with static provisioning instead of dynamic provisioning? I believe we discussed starting with static